### ISSUE TITLE
Run multiple test files at once

### ISSUE THREAD

**Issue by still-dreaming-1** [LucHermitte/vim-UT#1]
This plugin is great for when you first start writing tests. Running `UTRun %` and seeing the results in a quickfix window provides rapid feedback and a good workflow. But now I have multiple test files and I need to run each one separately, so this is no longer a good workflow. This is not too bad when I am just developing and I am mostly concerned with the tests from a single file at a time. But I want to run all the tests before I push my changes. Is there a way to run multiple test files at once? How should I organize my tests as they grow in size? I am using vim-UT to test my library plugin [vim-elhiv](https://github.com/still-dreaming-1/vim-elhiv) and my other plugin [vim-project-tags](https://github.com/still-dreaming-1/vim-project-tags)

**Comment by LucHermitte** [comment]
If you look closely at my plugins like [lh-cpp](https://github.com/LucHermitte/lh-cpp), you'll see that I'm able to run all my tests on travis -- thanks to helper scripts provided with vim-UT plus a few other projects. Running multiple tests is done thanks to rspec/rake. I can also do it in a console before commiting/pushing my modifications.

Moreover, `:UTRun` already takes several parameters. In lh-vim-lib, `:UTRun tests/lh/list.vim tests/lh/let.vim` works as expected. `UTRun tests/lh/*.vim` doesn't work as expected though. I'll have to look into this.

**Comment by still-dreaming-1** [comment]
@LucHermitte Thank you! That is so strange. I actually looked at the code for `:UTRun` before I posted this and saw that it takes multiple paramters. I tried passing multiple test files and it wasn't working for me before. But now it is. I guess it is a PEBKAC error. Being able to pass something like `tests/*.vim` option would be awesome! It would be great if it worked recursively, which I guess it would if it was just pattern matching.
